authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.1.7 on 2019-03-19 12:02
```

Comment:
@ManuelDominic , this miggration file is already exisitng in develop
Please remove it